web/src/compose_ui.js

```diff
@@ -68,7 +68,7 @@ export function smart_insert_inline($textarea, syntax) {
     autosize_textarea($textarea);
 }
 
-export function smart_insert_block($textarea, syntax) {
+export function smart_insert_block($textarea, syntax, padding_size = 2) {
```

Comment:
We might want to call this something like `padding_blank_lines=2` -- feels more readable to provide the units of "blank lines".

Reply:
Yeah, I was having a hard time coming up with something reasonable but not too long, thanks for this suggestion!

Reply:
Yeah, or maybe we should just call the parameter `padding_newlines`